### PR TITLE
Add eslint-plugin-mocha plugin

### PR DESCRIPTION
**Pull request by ndhoule**
@dominicbarnes

### CODE REVIEW

File: mocha/index.js [hunk 1]
```diff
@@ -5,8 +5,16 @@ module.exports = {
...
+  "plugins": [
+    "mocha"
+  ],
+
   "rules": {
     // Deeply nested functions are a reality of Mocha+BDD, so permit them
-    "max-nested-callbacks": 0
+    "max-nested-callbacks": 0,
+    "mocha/handle-done-callback": 2,
+    "mocha/no-exclusive-tests": 1,
```
Comment (dominicbarnes) [inline mocha/index.js]: Should this be an error instead of a warning? It could easily get past us on CI otherwise.

Comment (ndhoule) [inline mocha/index.js]: Totally, changing it now.

**Comment by ndhoule** [comment]
@dominicbarnes Updated, want to have another look?

**Comment by dominicbarnes** [comment]
lgtm!

**Comment by ndhoule** [comment]
@dominicbarnes thanks!

**Pull request merged**